### ISSUE TITLE
Unable to parse into HTML

### ISSUE THREAD

**Issue by ultiMEIGHT** [vimwiki/vimwiki#1281]
I am getting the following error when I run :VimwikiAll2HTML
```
Vimwiki: Saving Vimwiki files ...
Vimwiki: Deleting non-wiki html files ...
Vimwiki: Converting wiki to html files ...
Vimwiki: Processing /home/an4rki/Data/notes/Tasks.wiki
Error detected while processing function vimwiki#html#WikiAll2HTML[45]..<SNR>40_convert_file[110]..<SNR>40_get_html_
template:
line   20:
E484: Can't open file /home/an4rki/.local/share/nvim/site/pack/*/start/*/autoload/vimwiki/default.tpl
Press ENTER or type command to continue
```
Stable version 2.5

**Comment by dukenukem2d** [comment]
Hi, try this fix https://github.com/vimwiki/vimwiki/issues/1193#issuecomment-997033661.

**Comment by ultiMEIGHT** [comment]
> Hi, try this fix [#1193 (comment)](https://github.com/vimwiki/vimwiki/issues/1193#issuecomment-997033661).

Hi @dukenukem2d  thanks for replying. I actually came across this when I was searching for a fix but I was unable to understand how to actually fix it by looking at this issue. Could you please tell me which specific file needs changes and what changes in particular?

**Comment by ultiMEIGHT** [comment]
> > Hi, try this fix [#1193 (comment)](https://github.com/vimwiki/vimwiki/issues/1193#issuecomment-997033661).
> 
> Hi @dukenukem2d thanks for replying. I actually came across this when I was searching for a fix but I was unable to understand how to actually fix it by looking at this issue. Could you please tell me which specific file needs changes and what changes in particular?

Hey never mind, I managed to fix the issue according to #1193 like you mentioned. Thanks!
